README.md

```diff
@@ -41,3 +41,18 @@ CLI interface to our service
 ### `fission-server`
 
 Web server, database, 3rd party integrations
+
+# Building
+
+Fission is built inside of a pure Nix shell via the [Stack integration](https://docs.haskellstack.org/en/stable/nix_integration/). This means that you _should_ only need to type `stack build` to do a complete build of all packages.
+
+Unfortunately, the Stack integration doesn't play well with [`lorri`](https://github.com/target/lorri). If you've run `lorri` recently, you will either need to build with `--no-nix` or unset the `STACK_IN_NIX_SHELL` environment variable.
+
+```bash
+# bash or zsh
+unset STACK_IN_NIX_SHELL
+```
+
+```fish
+set -e STACK_IN_NIX_SHELL
+```
```

Comment:
This looks weird, and you're probably worried that `lorri` won't work. Well, `lorri` appears to be fine. Because the build is now pure, you don't need `lorri` to do the build itself — it's only for the other tools. It runs, but sets `STACK_IN_NIX_SHELL` (somehow). Unsetting that envar doesn't seem to effect lorri, and everything seems totally fine.

Reply:
I didn't have to do this, but it begs the question - do we still need lorri or should we drop it? we could also update .envrc to just`use_nix` ?

Reply:
So, we still want Lorri for other tooling. For example, if folks are going to run the server locally, they'll need Postgres. Having the language server, ghcide, and so on is also helpful so that getting an environment set up is as simple as `lorri shell`.

I kind of wonder about splitting out build from dev environment, but feels like overkill when it's ~5 or so tools.

> we could also update .envrc to just `use_nix`

Yeah, I'm actually unclear what `use_nix` gets us over lorri?

Reply:
Was just reading more about it after I wrote that... and I'm not sure? Other than lorri watches for changes. Yea, like i said, my build actually worked fine even with direnv + lorri so probably not worth making changes.

Although, if we're gonna change anything we should probably switch to https://github.com/nmattia/sorri because 🇨🇦